src/esbmc/bmc.cpp

```diff
@@ -946,6 +979,20 @@ smt_convt::resultt bmct::multi_property_check(
         }
       }
     }
+    else if (is_vb && result == smt_convt::P_UNSATISFIABLE)
```

Comment:
What is `is_vb`? What is `result`? can you use more intuitive variable names?

Reply:
Will change